source/neuropod/bindings/java/BUILD

```diff
@@ -95,8 +148,8 @@ java_library(
     data = [
         ":backends",
```

Comment:
This is no longer necessary (I was going to remove it in another PR, but I think it makes sense to do here). Can you also remove the definition of `backends` above please?